documentation/api/query/v4/operators.markdown

```diff
@@ -133,8 +133,33 @@ When only extracting a single column, the [] are optional
     ["extract" "transaction_uuid"
       ["=" "certname" "foo.com"]]
 
+When applying an aggregate function over a group-by clause, an extract
+statement takes the form
+
+    ["extract" [["function" "count"] "status"]
```

Comment:
We need commas in JSON :-). In fact the example above is lacks them also, and the null? example, yeah we're lousy with it :-).